heimctl/retention/scanner.go

```diff
@@ -21,6 +21,9 @@ func scanForExpired(ctx scope.Context, c cluster.Cluster, pb *psql.Backend) erro
 	rows, err := pb.DbMap.Select(
 		psql.Room{},
 		"SELECT name, founded_by, retention_days FROM room WHERE retention_days > 0")
+	if err == sql.ErrNoRows {
+		return nil
```

Comment:
This would be a good opportunity to reset some metrics: `roomHasExpiredMsg.Delete()`. Also, set lastExpiredScan so we don't get false alarms.